frontend/src/scenes/max/maxThreadLogic.tsx

```diff
@@ -669,8 +699,22 @@ export const maxThreadLogic = kea<maxThreadLogicType>([
         ],
 
         submissionDisabledReason: [
-            (s) => [s.formPending, s.question, s.threadLoading, s.activeStreamingThreads],
-            (formPending, question, threadLoading, activeStreamingThreads): string | undefined => {
+            (s) => [
+                s.formPending,
+                s.question,
+                s.threadLoading,
+                s.activeStreamingThreads,
+                s.deepResearchMode,
+                (s as any).deepResearchTemplate,
+            ],
+            (
+                formPending: boolean,
+                question: string,
+                threadLoading: boolean,
+                activeStreamingThreads: number,
+                deepResearchMode: boolean,
+                deepResearchTemplate: JSONContent | null
```

Comment:
I don't think we need to pass the entire template back to the API, the ID is enough?